### PR TITLE
feat: upgrade css-loader to v3

### PR DESCRIPTION
**Pull request by clarkdo**
### Breaking change

- @nuxt/webpack requires Node.js >= 8.9
- css-loader v3 breaking changes in options

### CODE REVIEW

File: packages/config/src/config/build.js [hunk 1]
```diff
@@ -41,7 +41,9 @@ export default () => ({
     },
     css: {},
     cssModules: {
-      localIdentName: '[local]_[hash:base64:5]'
+      modules: {
+        localIdentName: '[local]_[hash:base64:5]'
```
Comment (pi0) [inline packages/config/src/config/build.js]: Do you think is it good to normalize in `options.js` or make a warning about config breaking change?

Comment (clarkdo) [reply packages/config/src/config/build.js]: I think it would be better to keep consistency with css-loader, so maybe warning is better

Comment (clarkdo) [reply packages/config/src/config/build.js]: css-loader is a very basic loader, v3 breaking changes must have its consideration, so I think it’s better to align its behavior.
And it only affects users who have customized cssModues config, they should have a clear idea about what need to be changed

Comment (clarkdo) [reply packages/config/src/config/build.js]: BTW, css-loader has options schema validation 😺

**Comment by pi0** [comment]
@clarkdo is this PR ready to merge?

**Review by pi0 — APPROVED** [review]

**Pull request merged**